### ISSUE TITLE
Use "signposting" principles to make resources machine-friendly

### ISSUE THREAD

**Issue by bryjbrown** [Islandora/documentation#860]
Title (Goal) | Use "signposting" principles to make resources machine-friendly
-- | --
Primary Actor | Web crawlers & citation/annotation management tools
Scope | Drupal Module
Level | Low
Story | As a web crawler or citation/annotation management tool, I want to be able to accurately and consistently discern the PURL and license for a given resource, as well as discern the URLs of any associated objects or files via typed HTTP links, also known as [signposting](http://signposting.org/).

This would satisfy the following behaviours from the COAR NGR report:
- [Behaviour 1: Exposing Identifiers](http://ngr.coar-repositories.org/behaviour/exposing-identifiers/)
- [Behaviour 2: Declaring Licenses at the Resource Level](http://ngr.coar-repositories.org/behaviour/declaring-licenses-at-the-resource-level/)
- [Behaviour 3: Discovery through Navigation](http://ngr.coar-repositories.org/behaviour/discovery-through-navigation/)

**Comment by dannylamb** [comment]
Hey @bryjbrown, I'm assuming PURL is a Permanent URL?

Currently, we have all this except for pointing to the license, but CLAW currently provides signposts for all entity references, so if you modeling things a certain way you could probably get it without any programming.

**Comment by bryjbrown** [comment]
@dannylamb Yep, technically "persistant" URL, but you get the gist. Basically any URL that gets passed to a resolver, so ARKs, DOIs, Handles, etc are all examples of PURLs we'd want to expose via HTTP links.

I figured CLAW already did some of this stuff, I just wanted to make the use cases to show where we overlap with the COAR NGR report. Exposing the license shouldn't be too much work, right?
